src/app/contextBroker/contextBroker.cpp

```diff
@@ -745,10 +749,11 @@ static const char* validLogLevels[] =
   { "*",      ENT,          ENT_COMPS_V2,         ENT_COMPS_WORD,          badVerbGetPostOnly       }, \
                                                                                                        \
   { "GET",    IENT,         IENT_COMPS_V2,        IENT_COMPS_WORD,         getEntity                }, \
-  { "POST",   IENT,         IENT_COMPS_V2,        IENT_COMPS_WORD,         postEntity               }, \
-  { "PUT",    IENT,         IENT_COMPS_V2,        IENT_COMPS_WORD,         putEntity                }, \
+  { "GET",    IENTOA,       IENTOA_COMPS_V2,      IENTOA_COMPS_WORD,       getEntity                }, \
+  { "POST",   IENTOA,       IENTOA_COMPS_V2,      IENTOA_COMPS_WORD,       postEntity               }, \
+  { "PUT",    IENTOA,       IENTOA_COMPS_V2,      IENTOA_COMPS_WORD,       putEntity                }, \
   { "DELETE", IENT,         IENT_COMPS_V2,        IENT_COMPS_WORD,         deleteEntity             }, \
-  { "PATCH",  IENT,         IENT_COMPS_V2,        IENT_COMPS_WORD,         patchEntity              }, \
+  { "PATCH",  IENTOA,       IENTOA_COMPS_V2,      IENTOA_COMPS_WORD,       patchEntity              }, \
   { "*",      IENT,         IENT_COMPS_V2,        IENT_COMPS_WORD,         badVerbAllFive           }, \
                                                                                                        \
```

Comment:
IENTOA needs a 'badVerbXXX'. (should be tested as well in a functest)

Also, it seems that IENT and IENTOA are mixed.
Group all requests for one PATH and separate groups by an empty line (ending in backclash, of course). Easier to understand that way.

badVerbAllFive for IENT doean't seem correct anymore, as only GET and DELETE remain: badVerbGetDeleteOnly.